client/src/app/tabs/shared/fix-origin/FixOrigin.js

```diff
@@ -0,0 +1,58 @@
+var PADDING = 100;
```

Comment:
any chance to use `const` and `let` inside this module?

Reply:
I'll migrate the functionality to an external module (as a follow up step).

This will resolve the issue.